packages/blockchain/CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 7.0.1 - 2023-10-26
+
+### Dencun devnet-10 Compatibility
+
+This release contains various fixes and spec updates related to the Dencun (Deneb/Cancun) HF and is now compatible with the specs as used in [devnet-10](https://github.com/ethpandaops/dencun-testnet) (October 2023).
+
+- Update peer dependency for `kzg` module to use the official trusted setup for `mainnet`, PR [#3107](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3107)
+
+### Other Changes
+
+- New `getIteratorHeadSafe()` method not throwing if no iterator head is found, PR [#3099](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3099)
```

Comment:
I just checked but this is not correct (maybe the name of this method is misleading):

The difference between `getIteratorHead` and `getIteratorHeadSafe` is that: `getIteratorHead` returns the genesis block if the head itself is not found. (It *should* never throw). `getIteratorHeadSafe` returns `undefined` (not genesis block) if the head is not found. This is to account for edge cases where either safe/finalized block is not yet set by `forkchoiceUpdated` (then it is thus `undefined`), or `forkchoiceUpdated` sets this to genesis block (in that case, the `getIteratorHeadSafe` thus returns the genesis block, not `undefined`)